* allow bootsplash to work on non-vesa fb drivers
but check we have valid vc pointer.

Index: linux-2.6.16/drivers/video/bootsplash/bootsplash.c
===================================================================
--- linux-2.6.16.orig/drivers/video/bootsplash/bootsplash.c	2006-03-20 18:43:54.000000000 +0100
+++ linux-2.6.16/drivers/video/bootsplash/bootsplash.c	2006-03-20 18:46:53.000000000 +0100
@@ -548,11 +548,11 @@
 	    printk(KERN_INFO " found, but has oversized text area!\n");
 	    return -1;
 	}
-/*	if (!vc_cons[unit].d || info->fbops != &vesafb_ops) {
+	if (!vc_cons[unit].d) {
 	    splash_free(vc, info);
 	    printk(KERN_INFO " found, but framebuffer can't handle it!\n");
 	    return -1;
-	} */
+	}
 	printk(KERN_INFO "...found (%dx%d, %d bytes, v%d).\n", width, height, splash_size, version);
 	if (version == 1) {
 	    printk(KERN_WARNING "bootsplash: Using deprecated v1 header. Updating your splash utility recommended.\n");
